counterpoll/main.py

```diff
@@ -283,11 +283,17 @@ def _update_config_db(status, filename):
 
```

Comment:
@shlomibitton few comments here:
1. The name of the method `_update_config_db` doesn't reflect the actual implementation
2. No need to read a file if `FLEX_COUNTER_TABLE` is not present

Reply:
1 - Why not? this function is basically modify and save the config_db file.
2 - How can I know if the tables are not present in the file if I would not read it?

Reply:
@shlomibitton you are basically updating the `FLEX_COUNTER_TABLE` but not the entire `CONFIG_DB`. That's why it is not accurate. And for the second - yes, you are right.

Reply:
what about _update_config_db_flex_counter_table ?

Reply:
Done.